### ISSUE TITLE
04/22/2022 `npm` vulnerabilities 

### ISSUE THREAD

**Issue by phenggeler** [Recidiviz/pulse-dashboard#1847]
---
due: 2022-05-22
--- 
    
Please address the following vulnerabilities in this repository:

[async (npm)](https://github.com/Recidiviz/pulse-dashboard/security/dependabot)

**Comment by phenggeler** [comment]
@phenggeler - label applied: Due this month.

**Comment by phenggeler** [comment]
Hi @danawillow and @Catacola - just a reminder that this Dependabot alert has a due date of this month. In looking at the `async` write-up, I don't see that we are using the `mapValues` method call, but thought you all could verify!

**Comment by Catacola** [comment]
We only had two packages that wer both vulnerable and used in productions: `express-jwt` and `cahce-manager`.

I bumped `cache-manager` from `2.10.0` to `3.6.1` (the latest `2.x.x` was still vulnerable.

I bumped `express-jwt` from `6.0.0` to `6.1.2` which updated `async` to a patched version.

All of the other copied of it were used in `react-scripts` or `webdriverio` which are only used in build and testing, respectively.
